src/Services/src/CloudFoundry/CloudFoundryService.cs

```diff
@@ -687,15 +687,14 @@ public async Task<DetailedResult> DeleteAppAsync(CloudFoundryApp app, bool skipS
             };
         }
 
-        public async Task<DetailedResult> DeployAppAsync(CloudFoundryInstance targetCf, CloudFoundryOrganization targetOrg, CloudFoundrySpace targetSpace, string appName, string appProjPath, bool fullFrameworkDeployment, StdOutDelegate stdOutCallback, StdErrDelegate stdErrCallback, string manifestPath = null)
+        public async Task<DetailedResult> DeployAppAsync(CloudFoundryInstance targetCf, CloudFoundryOrganization targetOrg, CloudFoundrySpace targetSpace, string appName, string appProjPath, bool fullFrameworkDeployment, StdOutDelegate stdOutCallback, StdErrDelegate stdErrCallback, string stack, string manifestPath = null)
```

Comment:
At some point, at your discretion, when the number of args to a method/function exceeds some value, it might be worth creating a new object (simple struct would do) so that method only takes one (or close to one) arg.  The struct would have members for the args.  No need to change anything, just something to keep in back of mind.

Reply:
You read our minds! :) 
Yeah we were just thinking about doing exactly that; these arg lists are getting to be a pain to read. Maybe we'd call the struct something like `DeploymentConfig`?